app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,49 @@
+
+# ROUTES FOR COMMENTS ON QUESTIONS
+
+get '/questions/:id/comments' do
+  @question = Question.find(params[:id])
+  @comments = @question.comments
+  erb :'questions/comments/show'
+end
+
+get '/questions/:id/comments/new' do
+  @question = Question.find(params[:id])
+  erb :'comments/show'
+end
+
+post '/questions/:id/comments' do
+  if session_logged_in?
+    @question = Question.find(params[:id])
+    @author = session_current_user
+    @comment = @question.comments.create(body: params[:comment_body], author_id: @author.id, commentable_id: @question.id, commentable_type: "question" )
```

Comment:
Does the polymorphic association handle creating the commmentable_type and commmentable_id automatically?